MyApp/app/policies/setting_policy.rb

```diff
@@ -0,0 +1,27 @@
+class SettingPolicy < ApplicationPolicy
+  def index?
+    user.admin? && user.deleted_at == nil
+  end
+
+  def new?
+    user.admin?
+  end
+
+  def create? 
+    user.present? && user.deleted_at == nil && user.admin?
+  end
+
+  def update?
+    user.present? && user.admin? && user.deleted_at == nil
+  end
+
+  def destroy?
+    user.present? && user.admin? && user.deleted_at == nil
```

Comment:
why are you clling: user.present? somwhere, and somewhere not?
Im not saying this is the mistake, but what is the difference?

Reply:
There is no difference, I just forgot to add it in some methods. I will do that now.